### DISCUSSION TITLE
inline_message_id

### DISCUSSION THREAD

**Discussion by TheShubhendra** [python-telegram-bot/python-telegram-bot#2245] [Q&A] [answered]
How to get **inline_message_id** of game sent by user and set the game_score using it.

**Comment by Bibo-Joshi** [comment, marked as answer]
Hi. You get the `inline_message_id` of any inline result is through the corresponding `ChosenInlineResult`, which you can catch with the `ChosenInlineResultHandler`. The `CIR` will only have an `inline_message_id`, if there is a keyboard attached to the message, but I guess game messages always at least have the "play" button, so that shouldn't be a problem.
Note that you need to activate inline feedback with @BotFather to get those updates.
